graylog2-server/src/main/java/org/graylog2/indexer/searches/SearchesClusterConfig.java

```diff
@@ -63,6 +63,8 @@
             .add("file")
             .add("source_file")
             .build();
+    private static final Set<String> DEFAULT_LINKED_SEARCH_FIELDS = ImmutableSet.<String>builder()
```

Comment:
You could simply write `ImmutableSet.of()` or `Collections.emptySet()`.

Reply:
Changed!